release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -853,4 +860,30 @@ submodule openconfig-bgp-common {
   }
 
 
+  grouping auto-gen-link-bandwidth-config {
+    description
+      "Grouping of parameters related to automatic generation of
+      link-bandwidth extended community.";
+    leaf enabled {
+      description
+        "Generate link-bandwidth extnded community for BGP routes that do not
+        have link-bandwidth extnded community attached.";
+      type boolean;
+      default false;
+    }
+    leaf hold-down-time {
+      description
+        "Suppress frequent changes in the link-bandwidth value when bandwidth
+        increases.";
+      type uint16;
+    }
+    leaf transitive {
+      description
+        "Generated link-bandwidth attribute of transitive type. The non-transitive
```

Comment:
```suggestion
        "Generate link-bandwidth extended community as a transitive value. The non-transitive
```

Main suggestion here is to make sure that it's clear that we're generating the extcomm vs. a new attr.